controllers/main.go

```diff
@@ -164,6 +167,20 @@ func (controller *MainController) getNetworkName() string {
 	return controller.params.Name
 }
 
+// gets json data for current agendas from dcrdata.org
+func dcrDataAgendas() (agendasInfos []*dcrdatatypes.AgendasInfo, err error) {
+	resp, err := http.Get("https://explorer.dcrdata.org/api/agendas")
```

Comment:
Need to switch the URL for testnet.

We can use something like:

```
"https://" + controller.getNetworkName() + ".dcrdata.org"
```

Reply:
looks working!